jenkins/dockerfiles/build-and-push.sh

```diff
@@ -40,6 +40,7 @@ do
     d )  debos=true;;
     i )  dt_validation=true;;
     k )  k8s=true;;
+    Q )  qemu=true;;
```

Comment:
You need to add `Q` to the `options` variable and also in the comment describing each option.

Reply:
> and also in the comment describing each option.